scripts/leaf-scripts/scanalldbs.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * This file is used to get the different db versions after a deploy to verify update. 
+ * You will need to update the db version manually in this file to do the search at this time however

```

Comment:
Please update the filename and description to accurate reflect the purpose of this file.